pkg/v1/providers/infrastructure-vsphere/v1.0.1/ytt/overlay.yaml

```diff
@@ -109,7 +109,7 @@ kind: VSphereCluster
 metadata:
   name: #@ data.values.CLUSTER_NAME
 spec:
-  #@ if not data.values.AVI_CONTROL_PLANE_HA_PROVIDER:
+  #@ if data.values.AVI_CONTROL_PLANE_HA_PROVIDER != "true":
```

Comment:
can you change line 94 as well? `if (not data.values.AVI_CONTROL_PLANE_HA_PROVIDER)`

Reply:
There are many other places in the file using 
`not data.values.AVI_CONTROL_PLANE_HA_PROVIDER`
Do they also need to be changed?

Reply:
Hi Tom, after testing, it seems that we should treat `AVI_CONTROL_PLANE_HA_PROVIDER` as boolean. So no need to change this line. (correct way is still `if not AVI_CONTROL_PLANE_HA_PROVIDER`)